modules/atomic-widgets/assets/js/editor/utils/handle-duplicated-styles.js

```diff
@@ -0,0 +1,18 @@
+import { updateStyleId } from './update-style-id';
+import { getElementChildren } from './get-element-children';
+
+/**
+ * @typedef {import('assets/dev/js/editor/container/container')} Container
+ */
+
+/**
+ * Get a container - iterate over its children, find all styled atomic widgets and update their style ids
+ * @param {Container} duplicatedElement
+ */
+export function handleDuplicatedStyles( duplicatedElement ) {
+	const allElements = getElementChildren( duplicatedElement );
```

Comment:
Do you nee to split all those functions into small files or is it OK to put it on the same file?